src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +51,161 @@ export const KafkaPage: FunctionComponent = () => {
             <CardHeader>
               <CardHeaderMain>
                 <CardTitle>
-                  <Title headingLevel="h3">
-                    {t("kafkaoverview:purchaseCardTitle")}
-                  </Title>
+                  <Title headingLevel="h3">{t("purchaseCardTitle")}</Title>
                 </CardTitle>
               </CardHeaderMain>
             </CardHeader>
-            <CardBody>{t("kafkaoverview:purchaseCardMainText")}</CardBody>
+            <CardBody>{t("purchaseCardMainText")}</CardBody>
             <CardFooter>
               <Button
                 data-testid="cardPurchase-buttonCTA"
-                variant={ButtonVariant.secondary}
+                variant={ButtonVariant.link}
+                isLarge
+                isInline
                 component="a"
                 href="https://marketplace.redhat.com/en-us/products/red-hat-openshift-streams-for-apache-kafka"
                 target="_blank"
               >
-                {t("kafkaoverview:purchaseCardCallToActionButton")}
+                {t("purchaseCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
           <Card>
             <CardHeader>
               <CardTitle>
-                <Title headingLevel="h3">
-                  {t("kafkaoverview:contactSalesCardTitle")}
-                </Title>
+                <Title headingLevel="h3">{t("contactSalesCardTitle")}</Title>
               </CardTitle>
             </CardHeader>
-            <CardBody>{t("kafkaoverview:contactSalesCardMainText")}</CardBody>
+            <CardBody>{t("contactSalesCardMainText")}</CardBody>
             <CardFooter>
               <Button
                 data-testid="cardContactSales-buttonCTA"
-                variant={ButtonVariant.secondary}
+                variant={ButtonVariant.link}
+                isLarge
+                isInline
                 component="a"
-                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-sales"
+                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-us"
                 target="_blank"
               >
-                {t("kafkaoverview:contactSalesCardCallToActionButton")}
+                {t("contactSalesCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
         </Grid>
       </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("pricingModalTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex>
+              <Flex flex={{ default: "flex_1" }}>
```

Comment:
The mockup doesn't clearly show this, but I _think_ the intention is to have these contents (i.e. first section in Pricing model) centered, both vertically and horizontally. 

You can achieve this by adding the following props to this second `<Flex>` component:
`alignSelf={{ default: 'alignSelfCenter'}} justifyContent={{ default: 'justifyContentCenter' }}`